_community/google-groups.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Google Groups - Managing List Membership
+layout: text-optimized-width
+
+---
+
+# {{ page.title }}
+
+This documents how to join and leave a Google Group via webpage and email.
+
+If you have any issues following the guides below please reach out to us at [support@osg-htc.org](mailto:support@osg-htc.org) with the email you would like subscribed and the list you would like to subscribe too.
+
+##### Contents
+
+- [Joining a list via webpage](#joining-a-list-via-webpage)
+- [Leaving a list via webpage](#leaving-a-list-via-webpage)
+- [Joining a list via email](#joining-a-list-via-email)
+- [Leaving a list via email](#leaving-a-list-via-email)
+
+## Joining a list via webpage
+
+To join a group via the Google Groups UI you should:
+
+1. Login to google account you wish to subscribe.
```

Comment:
1. Choose an email address to subscribe
2. If that email address is associated with a Google Account, then… (and skip to Step N when done)
3. If not, then…

Reply:
I went a slightly different direction with this and decided to fork them after the first step into two different methods. Neither of the method share ending steps so it made sense to just completely separate them.